mongo_connector/connector.py

```diff
@@ -520,6 +524,11 @@ def apply_logging(option, cli_values):
         '--stdout', dest='stdout', action='store_true', help=
         'Log all output to STDOUT rather than a logfile.')
 
+    logging.add_cli(
+        "--logformat", dest="log_format", help=
```

Comment:
Most of the command-line options have dashes that separate words, so I think this should be `--log-format`.